public/app/features/panel/components/VizTypePicker/VisualizationSuggestionCard.tsx

```diff
@@ -88,13 +96,34 @@ const getStyles = (theme: GrafanaTheme2) => {
         background: ${theme.colors.background.secondary};
       }
     `,
+    imgBox: css`
+      display: flex;
+      flex-direction: column;
+      height: 100%;
+
+      justify-self: center;
+      color: ${theme.colors.text.primary};
+      width: 100%;
+      display: flex;
+
+      justify-content: center;
+      align-items: center;
+      text-align: center;
+    `,
     name: css`
+      padding: ${theme.spacing(1)};
       font-size: ${theme.typography.bodySmall.fontSize};
       white-space: nowrap;
       overflow: hidden;
       color: ${theme.colors.text.secondary};
       font-weight: ${theme.typography.fontWeightMedium};
       text-overflow: ellipsis;
+      padding-bottom: ${theme.spacing(0.5)};
+      margin-top: ${theme.spacing(-2)};
```

Comment:
i'm not a huge fan of negative margins tbh. if the duplicate `padding` on 114 is removed everything looks fine to me without it. wdyt?

Reply:
me neither but what you suggest is not the same, want the image to be perfectly centered and it's not unless I push it back up to account for the space the heading/name text takes